Makefile.am

```diff
@@ -24,255 +25,122 @@
-    test/integration/create-keyedhash-sha1-hmac.int \
-    test/integration/encrypt-decrypt.int \
-    test/integration/encrypt-decrypt-2.int \
-    test/integration/evict-ctrl.int \
-    test/integration/get-random.int \
-    test/integration/hierarchy-change-auth.int \
-    test/integration/abi-version.int \
-    test/integration/pcr-extension.int \
-    test/integration/self-test.int \
-    test/integration/start-auth-session.int \
-    test/integration/sys-initialize.int \
-    test/integration/tpm-properties.int \
-    test/integration/system-api.int \
-    test/integration/policy-template.int \
-    test/integration/create-loaded.int \
-    test/integration/policy-authorizeNV.int \
-    test/tpmclient/tpmclient.int
-if ESAPI
-TESTS_INTEGRATION += \
-    test/integration/esys-clockset.int \
-    test/integration/esys-clockset-session.int \
-    test/integration/esys-create-fail.int \
-    test/integration/esys-create-password-auth.int \
-    test/integration/esys-create-primary-ecc-hmac.int \
-    test/integration/esys-create-primary-hmac.int \
-    test/integration/esys-create-session-auth.int \
-    test/integration/esys-evict-control-serialization.int \
-    test/integration/esys-get-capability.int \
-    test/integration/esys-get-random.int \
-    test/integration/esys-make-credential.int \
-    test/integration/esys-make-credential-session.int \
-    test/integration/esys-nv-ram-extend-index.int \
-    test/integration/esys-nv-ram-extend-index-session.int \
-    test/integration/esys-nv-ram-ordinary-index.int \
-    test/integration/esys-nv-ram-ordinary-index-session.int \
-    test/integration/esys-quote.int \
-    test/integration/esys-rsa-encrypt-decrypt.int \
-    test/integration/esys-save-and-load-context.int
-endif #ESAPI
-endif #SIMULATOR_BIN
 
-CLEANFILES = \
-    AUTHORS \
-    $(nodist_pkgconfig_DATA) \
-    $(man3_MANS) \
-    $(man7_MANS) \
-    test/integration/*.log \
-    test/tpmclient/*.log \
-    test/unit/*.log
 
-# headers and where to install them
-libsapidir      = $(includedir)/sapi
-libsapi_HEADERS = $(srcdir)/include/sapi/*.h
-libtctidir      = $(includedir)/tcti
-libtcti_HEADERS = $(srcdir)/include/tcti/*.h
-if ESAPI
-libesapidir      = $(includedir)/esapi
-libesapi_HEADERS = $(srcdir)/include/esapi/*.h
-endif
+### Add ax_* rules ###
+# ax_code_coverage
+@CODE_COVERAGE_RULES@
+
+# ax_doxygen
+@DX_RULES@
+MOSTLYCLEANFILES += $(DX_CLEANFILES)
+
+
+
+### Include make files ###
+# Add source code files from bootstrap
+include src_vars.mk
+
+# Add test definitions
+include Makefile-test.am
+
 
-# pkg-config files
-pkgconfigdir          = $(libdir)/pkgconfig
-nodist_pkgconfig_DATA = \
-    lib/marshal.pc \
-    lib/sapi.pc \
-    lib/tcti-device.pc \
-    lib/tcti-socket.pc
-# man pages / documentation
-man3_MANS = man/man3/InitDeviceTcti.3 man/man3/InitSocketTcti.3
-man7_MANS = man/man7/tcti-device.7 man/man7/tcti-socket.7
 
-EXTRA_DIST = \
-    AUTHORS \
+### Distributiuon files ###
+# EXTRA_DIST Files
```

Comment:
I'm not sure these comments add any useful data. If you think the EXTRA_DIST variable needs documentation we should add what it does: include these files in the release tarball.